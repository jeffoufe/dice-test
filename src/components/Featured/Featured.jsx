// @flow

import React from 'react';
import styles from './Featured.module.scss';

const Featured = () => (
    <div className={styles.container}>
        <div className={styles.featured}>featured</div> 
    </div>
)

export default Featured;