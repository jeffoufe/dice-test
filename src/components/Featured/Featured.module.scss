@import "../../constants/colors";

.container {
    width: 81px;
    padding: 2px 5px 4px;
    font-weight: 600;
    position: absolute;
    height: 23px;
    right: 17px;
    bottom: 17px;
    font-size: 14px;
    line-height: 16px;
    letter-spacing: 0.05em;
    text-transform: uppercase;
    color: white;
    background-color: $blue;
}

.featured {
    display: inline-block;
    vertical-align: -webkit-baseline-middle;
}