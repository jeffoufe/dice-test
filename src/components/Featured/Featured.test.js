import Featured from './Featured';
import { shallow } from 'enzyme';
import React from 'react';

describe('Featured', () => {
    const wrapper = shallow(
        <Featured />
    );

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })
})