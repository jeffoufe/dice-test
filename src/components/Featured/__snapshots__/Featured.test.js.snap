// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Featured renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="featured"
  >
    featured
  </div>
</div>
`;
